Update2 iq.m3u

Redo-San opened the pull request:
I added more channels and verified that they are working and deleted some closed channels and updated the links of some channels as well

Redo-San committed:
Update iq.m3u

I added Al Rabiaa TV channel and modified the link of Dijlah TV channel

Redo-San committed:
Update iq.m3u

Redo-San committed:
Update iq.m3u

Redo-San committed:
I added more channels and verified that they are working and deleted some closed channels and updated the links of some channels as well

UltraHDR commented on streams/iq.m3u:
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlGhadTV.eg",Al Ghad (720p) [Not 24/7]
```
Only iraq channels are added to iq.m3u. Please remove channels with a tvg-id that does not end with `.iq` and add them to the playlist for that country.

Redo-San replied:
Ok

UltraHDR commented on streams/iq.m3u:
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
+#EXTINF:-1 tvg-id="AlNaeemTV.iq",Al-Naeem TV (576p)
+https://nl2.livekadeh.com/hls2/alnaeem_high.m3u8
+#EXTINF:-1 tvg-id="alrabiaa.tv",Al Rabiaa TV (1080p)
+https://cllive.itworkscdn.net/alrabiaatvnewslive/token=nva=1690991331~dirs=1~hash=058a79456eeadb3deb21b/alrabiaatv.smil/playlist.m3u8
```
Expires

kaikyoofers replied:
@UltraHDR you didnt check there was a incorrect tvg-id.

UltraHDR commented on streams/iq.m3u:
```diff
@@ -5,14 +5,34 @@ http://rtmp1.abnsat.com/hls/arabic.m3u8
+#EXTINF:-1 tvg-id="AlMayadeenTV.lb",Al Mayadeen (576p)
+https://mdnlv.cdn.octivid.com/almdn/smil:mpegts.stream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AlNaeemTV.iq",Al-Naeem TV (576p)
+https://nl2.livekadeh.com/hls2/alnaeem_high.m3u8
```
```suggestion
https://nl2.livekadeh.com/hls2/alnaeem.m3u8
```

UltraHDR committed:
Update streams/iq.m3u

UltraHDR commented on streams/iq.m3u:
```diff
@@ -29,16 +49,46 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-https://ghaasiflu.online/Dijlah/index.m3u8
```
Works

UltraHDR commented on streams/iq.m3u:
```diff
@@ -29,16 +49,46 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
 #EXTINF:-1 tvg-id="AssyrianANB.iq",Assyrian ANB (720p)
 https://anbsat.ddns.net/anb/live/ANB/playlist.m3u8
-#EXTINF:-1 tvg-id="DijlahTV.iq",Dijlah TV (1080p)
-http://91.134.145.75:10001/Dijlah/index.m3u8
```
Works

UltraHDR replied:
Please add it back

Redo-San replied:
> Please add it back

It's an old link and there are always interruptions
The new link I added is much better

Redo-San committed:
Update iq.m3u

I've removed channels that don't have an iq identifier
And removed channels with expired links

UltraHDR commented on streams/iq.m3u:
```diff
@@ -29,16 +29,32 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
+#EXTINF:-1 tvg-id="BeitolAbbasTVChannel.iq",BeitolAbbas TV Channel (720p)
+https://live.beitolabbas.tv/live/beitolabbastv.m3u8
+#EXTINF:-1 tvg-id="dijlah.tv",Dijlah TV (1080p)
+https://ghaasiflu.online/Dijlah/tracks-v1a1/mono.m3u8
```
```suggestion
https://ghaasiflu.online/Dijlah/index.m3u8
```

UltraHDR committed:
Update streams/iq.m3u

UltraHDR commented on streams/iq.m3u:
```diff
@@ -29,16 +29,32 @@ https://5d94523502c2d.streamlock.net/home/mystream/playlist.m3u8
+#EXTINF:-1 tvg-id="dijlah.tv",Dijlah TV (1080p)
 https://ghaasiflu.online/Dijlah/index.m3u8
+#EXTINF:-1 tvg-id="DuaChannel.iq",Dua Channel (720p)
+https://live.ishiacloud.com/haditv.co.uk/dua-channel.m3u8
```
Not working

Redo-San replied:
But the link works when I run it?

UltraHDR replied:
It works in mpv but not vlc so I think it's ok to have it

UltraHDR commented on streams/iq.m3u:
```diff
@@ -65,6 +79,16 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="Kurdsat.iq",Kurdsat (1080p)
+https://ikomg2.mmdlive.lldns.net/ikomg2/366ca61db4c646a2bf9cd9f92dd6321c/manifest.m3u8
+#EXTINF:-1 tvg-id="KurdsatNews.iq",KurdSat News (1080p) [Not 24/7]
+https://ikomg2.mmdlive.lldns.net/ikomg2/107b7df8f5444d778f349100739a09cd/manifest.m3u8
```
already there on line 79

UltraHDR commented on streams/iq.m3u:
```diff
@@ -3,16 +3,16 @@
-https://sd343444.vodu.store:3356/live/Alghadeer/index.m3u8
+#EXTINF:-1 tvg-id="AlforatTV.iq",Alforat TV (1080p) 
+http://95.216.180.111:1935/live/10/playlist.m3u8
+#EXTINF:-1 tvg-id="AlhurraIraq.us",Alhurra Iraq (720p)
```
Should be in the US.m3u playlist

Redo-San replied:
No, it is an Iraqi channel, speaking in Arabic, but it broadcasts from inside America, so it is not an American channel.

UltraHDR replied:
It's based in the US
https://en.wikipedia.org/wiki/Alhurra

UltraHDR commented on streams/iq.m3u:
```diff
@@ -99,19 +123,19 @@ https://ythls.onrender.com/UCi4CSgJk3i_slWnWG9Y14qA.m3u8
 https://live.ishiacloud.com/haditv.co.uk/dua-channel.m3u8
 #EXTINF:-1 tvg-id="IshtarTV.iq",Ishtar TV (1080p) 
 http://ishtar.cdncast.xyz:1935/live/iShtarHD/playlist.m3u8
+#EXTINF:-1 tvg-id="InewsTV.iq",Inews TV (1080p)
```
Duplicate. already there on line 71

Redo-San committed:
Update4 iq.m3u

I deleted the duplicate channels, and also deleted the closed channels, and arranged the channels according to the alphabet.
Tell me if there are any errors. Thanks🌹

UltraHDR commented on streams/iq.m3u:
```diff
@@ -65,53 +83,35 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
+https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
+https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
```
Duplicate link

UltraHDR commented on streams/iq.m3u:
```diff
@@ -65,53 +83,35 @@ https://5a3ed7a72ed4b.streamlock.net/live/SMIL:myStream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="ZagrosTV.iq",Zagros (720p) [Not 24/7]
-https://5a3ed7a72ed4b.streamlock.net/zagrostv/SMIL:myStream.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="NUBARPlusPlus.iq",NUBAR Plus TV (720p)
-http://stream.nubar.tv:1935/private/NUBARPlus/playlist.m3u8
```
works

Redo-San committed:
Update5 iq.m3u

I deleted the two duplicate channels, and I also deleted the Ishtar channel, which has been off the air for almost three years.

UltraHDR committed:
Update iq.m3u

UltraHDR committed:
Update us.m3u

UltraHDR committed:
Update iq.m3u

UltraHDR committed:
Update iq.m3u

UltraHDR committed:
Update us.m3u

UltraHDR committed:
Update iq.m3u

BellezaEmporium reviewed (approved):
In case you are wondering : 

http://95.216.180.111:1935/live/1/playlist.m3u8 <- AFAQ TV
http://95.216.180.111:1935/live/2/playlist.m3u8 <- could be Al Ghadeer TV
http://95.216.180.111:1935/live/3/playlist.m3u8 <- Al Noujaba
http://95.216.180.111:1935/live/4/playlist.m3u8 <- Al Sharqiya News
http://95.216.180.111:1935/live/6/playlist.m3u8 <- Al Ayyam TV
http://95.216.180.111:1935/live/7/playlist.m3u8 <- Al Ahd
http://95.216.180.111:1935/live/8/playlist.m3u8 <- Al Arabiya
http://95.216.180.111:1935/live/9/playlist.m3u8 <- Al Hadath
http://95.216.180.111:1935/live/11/playlist.m3u8 <- Alquds Satellite Channel
http://95.216.180.111:1935/live/12/playlist.m3u8 <- Pal Today TV
http://95.216.180.111:1935/live/13/playlist.m3u8 <- Al Anwar 2
http://95.216.180.111:1935/live/14/playlist.m3u8 <- ?
![snapshot4](https://github.com/iptv-org/iptv/assets/30985701/f7b60f1a-b87c-4907-8406-045b95f4067e)

(in case it goes down, screenshot above)

http://95.216.180.111:1935/live/15/playlist.m3u8 <- ifilm Farsi
http://95.216.180.111:1935/live/16/playlist.m3u8 <- Thaqalayn TV
http://95.216.180.111:1935/live/17/playlist.m3u8 <- ?
![snapshot5](https://github.com/iptv-org/iptv/assets/30985701/2c98a714-005a-49ba-94bd-babbf81dd42f)

(in case it goes down, screenshot above)
http://95.216.180.111:1935/live/18/playlist.m3u8 <- Al Janoub
http://95.216.180.111:1935/live/19/playlist.m3u8 <- Al Eshraq TV
http://95.216.180.111:1935/live/20/playlist.m3u8 <- Nabaa TV
http://95.216.180.111:1935/live/21/playlist.m3u8 <- Press TV English
http://95.216.180.111:1935/live/22/playlist.m3u8 <- Al Naeem
http://95.216.180.111:1935/live/23/playlist.m3u8 <- Voice of America Farsi 365
http://95.216.180.111:1935/live/24/playlist.m3u8 <- from Google's own translation : Peace TV
http://95.216.180.111:1935/live/28/playlist.m3u8 <- BBC Persian
http://95.216.180.111:1935/live/29/playlist.m3u8 <- CNN International
http://95.216.180.111:1935/live/30/playlist.m3u8 <- from Google's own translation : Al Ittihad
http://95.216.180.111:1935/live/33/playlist.m3u8 <- Al Etejah TV
Server name : 63b03f7689049.streamlock.net
